less/_defaults/_font-mixins.less

```diff
@@ -0,0 +1,245 @@
+.page-title() {
+  font-family: @page-title-family;
+  font-size: @page-title-size;
+  font-weight: @page-title-weight;
+  line-height: @page-title-line-height;
+}
+
+.page-subtitle() {
+  font-family: @page-subtitle-family;
+  font-size: @page-subtitle-size;
+  font-weight: @page-subtitle-weight;
+  line-height: @page-subtitle-line-height;
+}
+
+.page-body() {
+  font-family: @page-body-family;
+  font-size: @page-body-size;
+  font-weight: @page-body-weight;
+  line-height: @page-body-line-height;
+}
+
+.page-instruction() {
+  font-family: @page-instruction-family;
+  font-size: @page-instruction-size;
+  font-weight: @page-instruction-weight;
+  line-height: @page-instruction-line-height;
+  font-style: italic;
+}
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Page
+// --------------------------------------------------
+.article-title() {
+  font-family: @article-title-family;
+  font-size: @article-title-size;
+  font-weight: @article-title-weight;
+  line-height: @article-title-line-height;
+}
+
+.block-title() {
+  font-family: @block-title-family;
+  font-size: @block-title-size;
+  font-weight: @block-title-weight;
+  line-height: @block-title-line-height;
+}
+
+.component-title() {
+  font-family: @component-title-family;
+  font-size: @component-title-size;
+  font-weight: @component-title-weight;
+  line-height: @component-title-line-height;
+}
+
+.notify-title() {
+  font-family: @notify-title-family;
+  font-size: @notify-title-size;
+  font-weight: @notify-title-weight;
+  line-height: @notify-title-line-height;
+}
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Drawer
+// --------------------------------------------------
+.drawer-item-title() {
+  font-family: @drawer-item-title-family;
+  font-size: @drawer-item-title-size;
+  font-weight: @drawer-item-title-weight;
+  line-height: @drawer-item-title-line-height;
+}
+
+.drawer-item-body() {
+  font-family: @drawer-item-body-family;
+  font-size: @drawer-item-body-size;
+  font-weight: @drawer-item-body-weight;
+  line-height: @drawer-item-body-line-height;
+}
+
+.resources-filter-btn() {
+  font-family: @resources-filter-btn-family;
+  font-size: @resources-filter-btn-size;
+  font-weight: @resources-filter-btn-weight;
+  line-height: @resources-filter-btn-line-height;
+}
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Components
+// --------------------------------------------------
+.item-title() {
+  font-family: @item-title-family;
+  font-size: @item-title-size;
+  font-weight: @item-title-weight;
+  line-height: @item-title-line-height;
+}
+
+.pull-quote-title() {
+  font-size: @pull-quote-title-size;
+  font-family: @pull-quote-title-family;
+  font-weight: @pull-quote-title-weight;
+  line-height: @pull-quote-title-line-height;
+}
+
+.pull-quote-body() {
```

Comment:
I've notice we don't currently include any styling for `<blockquote>` although this element is available in the Authoring tool text editor. 
The pull quote mixins have been added with the intention of styling text components to enhance the text but for accessibility we should probably be using `<blockquote>` where appropriate.

Not sure if this is the best place for it but could we add the following please?

```
blockquote {
  .pull-quote-body();
}
```